### DISCUSSION TITLE
VRP and road traffic [python]

### DISCUSSION THREAD

**Discussion by ivandorte** [TimefoldAI/timefold-solver#925] [General]
A simple informative question:  Does the VRP solver use road traffic data like TomTom, Google, Here?
Thanks!!!
Ivan

**Comment by ge0ffrey** [comment]
Yes.

In Timefold Community, you can edit the Location class to add a distance matrix or travel time matrix yourself.

Note that running maps calculation locally involves dealing with large GIS data files and cpu limitations.


In Timefold Enterprise (work in progress), we handle the orchestration and compute challenges for you. It just appears in the Location class. By default we use OpenStreetMap data, but we can connect to other services too.
